dockerfile_scripts/ompi.sh

```diff
@@ -2,11 +2,13 @@
 
 set -x
 
-# Install the Mellanox OFED stack.  Note that this is dependent on what 
+# Install the Mellanox OFED stack.  Note that this is dependent on what
 # the base OS is (ie, Ubuntu 20.04) so if that changes then this needs updated.
 #MOFED_VER=5.0-2.1.8.0
-MOFED_VER=5.5-1.0.3.2
+#MOFED_VER=5.5-1.0.3.2
+MOFED_VER=5.4-3.4.0.0
```

Comment:
Is there a reason you’re using 5.4 instead of 5.5?

Reply:
I tried to wget 5.5 and it couldn't find it so I tried 5.4 and that worked.